src/assets/toolkit/scripts/home-animation.js

```diff
@@ -0,0 +1,299 @@
+/**
+ * GSAP is powerful and performant, but its organization is a little old-school.
+ * This imports a bunch of functions to the global scope.
+ *
+ * See: https://github.com/greensock/GreenSock-JS/issues/71
+ */
+
+import TweenLite from 'gsap/src/uncompressed/TweenLite';
+import TimelineMax from 'gsap/src/uncompressed/TimelineMax';
+import EasePack from 'gsap/src/uncompressed/easing/EasePack';
+import CSSPlugin from 'gsap/src/uncompressed/plugins/CSSPlugin';
+import AttrPlugin from 'gsap/src/uncompressed/plugins/AttrPlugin';
+
+/**
+ * Settings
+ */
+
+const settings = {
+  morph: {
+    delay: '+=1',
+    duration: 0.5,
+    ease: Elastic.easeOut.config(0.5, 0.4)
+  }
+};
+
+/**
+ * Timelines
+ */
+
+const timelines = {
+  morph: new TimelineMax({
+    repeat: -1
+  })
+};
+
+/**
+ * Elements
+ */
+
+const dom = (function (nodeList) {
+  const result = {};
+
+  for (let i = 0; i < nodeList.length; i++) {
+    let node = nodeList[i];
+    let id = node.getAttribute('id');
+
+    if (id) {
+      result[id] = node;
+    }
+  }
+
+  return result;
+})(document.querySelectorAll('*[id]'));
+
+TweenLite.set('.js-scaleIn', {
+  visibility: 'visible',
+  scale: 0,
+  transformOrigin: '50% 50%'
+});
+
+/**
+ * Device morphing
+ */
+
+timelines.morph.addLabel('toTablet', settings.morph.delay);
+
+timelines.morph.to(dom.bezel, settings.morph.duration, {
+  attr: {
+    x: 94,
+    y: 66,
+    width: 132,
+    height: 188
+  },
+  ease: settings.morph.ease
+}, 'toTablet');
```

Comment:
Could instances, such as `'toTablet'`, be pulled out into a `const` setting object maybe?